playwright-e2e/lib/component/Question.ts

```diff
@@ -7,13 +7,15 @@ export default class Question {
   private readonly locator: Locator;
   private readonly rootLocator: Locator;
 
-  constructor(page: Page, opts: { prompt: string | RegExp; parentSelector?: Locator }) {
-    const { prompt, parentSelector } = opts;
+  constructor(page: Page, opts: { prompt?: string | RegExp; classAttr?: string; parentSelector?: Locator }) {
```

Comment:
Some UI fields don't have `data-ddp-test` property, but they do have a class attribute which is unique and can be used in css selectors.